lib/packages/shared-types/opensearch/main/transforms/seatool.ts

```diff
@@ -134,12 +134,16 @@ export const transform = (id: string) => {
       ? SEATOOL_SPW_STATUS[data.STATE_PLAN.SPW_STATUS_ID]
       : "Unknown";
     const { stateStatus, cmsStatus } = getStatus(seatoolStatus);
+
+    //const capitalizeId = (id) => typeof id === "string" ? id.charAt(0).toUpperCase() + id.slice(1) : "";
+    const capitalizeLastThree = (id: string): string => id.replace(/([a-z]{3})$/, (match) => match.toUpperCase());
+
     const resp = {
-      id,
+      id: capitalizeLastThree(id),
```

Comment:
```suggestion
      id: id.toUpperCase(),
```
I think this is all you need to do. You shouldn't need that function.